package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pwabuilder-macos",
-  "version": "0.3.9",
+  "version": "0.3.10",

```

Comment:
Can you bump API to use this new pwabuilder-macos? And bump the web app to use the new API?